package.json

```diff
@@ -114,7 +114,9 @@
       "jest --maxWorkers=1 --bail --findRelatedTests"
     ]
   },
-  "dependencies": {},
+  "dependencies": {
+    "shelljs": "^0.8.3"
```

Comment:
This shouldn't be in the dependencies, should be moved to `devDependencies`

Reply:
Good point. It's been moved.